app/controllers/animal_selector.coffee

```diff
@@ -72,7 +72,13 @@ class AnimalSelector extends Controller
         targets = @items.filter ':not(".dimmed")'
         targets.first().focus()
         targets.first().click() if targets.length is 1
-        AnalyticsLogger.logEvent 'search', @searchInput.val()
+        Geordi.logEvent {
+          type: 'identify'
```

Comment:
good spot - it's supposed to be "search". Copy and paste error, will fix. Exactly what code review is good for!!